shell/components/ProgressBarMulti.vue

```diff
@@ -122,6 +122,7 @@ function toPercent(value, min, max) {
     v-trim-whitespace
     :class="{progress: true, multi: pieces.length > 1}"
     :aria-label="ariaLabelText"
+    role="group"
```

Comment:
Would `role="progressbar"` be a better fit for this case?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/progressbar_role

Reply:
fixed